v3/src/components/case-table/collection-table.tsx

```diff
@@ -1,5 +1,8 @@
 import React, { useRef } from "react"
 import DataGrid, { DataGridHandle } from "react-data-grid"
+// eslint wants us to add @types/pluralize to our deps instead of devDependencies
+// Are we okay with moving types into our main deps, or do we just want to suppress this warning?
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I did not encounter any lint issues when I moved it to `devDependencies` in my PR. Types are logically `devDependencies` because they're not relevant at run-time. My preference would be to leave it in `devDependencies` and figure out why the linter is behaving differently for you than for me.